### ISSUE TITLE
issue: 400 Components: Data was not an array

### ISSUE THREAD

**Issue by imsergiobernal** [gioboa/jira-plugin#53]
**Describe the issue**
Trying to create an issue, I'm getting this error

Error: {"statusCode":400,"body":{"errorMessages":[],"errors":{"components":"data was not an array"}}...

If I dont set any component, works perfectly.

**To Reproduce**
1. Try to create an issue
2. Select a component
3. Send it

**Expected behavior**
Create an issue

**Comment by gioboa** [comment]
Hi @imsergiobernal,
mmm seems that the plugin don't set correctly the components field...
If you open this url in your browser
https://_<your_jira_baseurl>_/rest/api/2/issue/createmeta?expand=projects.issuetypes.fields
you can retrive all the fields for each project/type in json format.
(you must be logged in in the web application)
Now if you copy and paste the json here -> https://jsoneditoronline.org/ you can see the data in readable way.
Can you post here all the required fields for your project/type? Thanks 👍
----
Please could you change you [review](https://marketplace.visualstudio.com/items?itemName=gioboa.jira-plugin#review-details)? Thanks. 👍

**Comment by imsergiobernal** [comment]
Changed! I'm not able to post that :( I see a lot of possible sensible info there

**Comment by gioboa** [comment]
Ok don't worry.
Could you post only components field?
You can tarnish all the sensitive data. 👍
For me it's important understand where I can retrieve all the available values for that field.

**Comment by imsergiobernal** [comment]
There you have @gioboa ! 

```json
{
  "components": {
    "required": false,
    "schema": {
      "type": "array",
      "items": "component",
      "system": "components"
    },
    "name": "Component/s",
    "key": "components",
    "hasDefaultValue": false,
    "operations": [
      "add",
      "set",
      "remove"
    ],
    "allowedValues": [
      {
        "self": "https://-.atlassian.net/rest/api/2/component/1",
        "id": "1",
        "name": "A"
      },
      {
        "self": "https://-.atlassian.net/rest/api/2/component/2",
        "id": "2",
        "name": "B"
      },
      {
        "self": "https://-.atlassian.net/rest/api/2/component/3",
        "id": "3",
        "name": "C"
      },
      {
        "self": "https://-.atlassian.net/rest/api/2/component/4",
        "id": "4",
        "name": "D"
      }
    ]
  }
}
```

**Comment by gioboa** [comment]
@imsergiobernal 
I published [v 0.11.4](https://github.com/gioboa/jira-plugin/releases/tag/0.11.4)
I hope solve your issue.
Please let me know.
